composer.json

```
@@ -15,7 +15,8 @@
     "johnpbloch/wordpress": "5.4.2",
     "altis/cms-installer": "^0.4.2",
     "johnbillion/extended-cpts": "^4.4.0",
-    "humanmade/clean-html": "^2.0.0"
+    "humanmade/clean-html": "^2.0.0",
+    "humanmade/post-cloner": "1.1.1"
```

```suggestion
    "humanmade/post-cloner": "~1.1.1"
```

Let's track the patch release version